### ISSUE TITLE
Invalid code example for HttpServer with HTTP/2 in reference guide

### ISSUE THREAD

**Issue by sephiroth-j** [reactor/reactor-netty#1225]
[The listing with the example code for an HTTP Server with HTTP/2 in chapter 5.8](https://projectreactor.io/docs/netty/release/reference/index.html#_http2) is invalid because the SSL context of the `SslContextSpec` is not set.

**actual**
```java
                          .secure(spec ->
                                  SslContextBuilder.forServer(new File("certificate.crt"),
                                                              new File("private.key")))
```

**expected**
```java
                          .secure(spec ->
                                  spec.sslContext(SslContextBuilder.forServer(new File("certificate.crt"),
                                                              new File("private.key"))))
```

**Comment by violetagg** [comment]
@sephiroth-j Would you like to provide a PR?

**Comment by sephiroth-j** [comment]
Sure.

**Comment by violetagg** [comment]
Fixed with #1226
